.github/workflows/psalm.yml

```diff
@@ -0,0 +1,61 @@
+name: Static analysis
+
+on:
+  pull_request: ~
+
+jobs:
+  psalm:
+    name: Psalm
+    runs-on: Ubuntu-20.04
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+          extensions: "json,couchbase,memcached,mongodb,redis,xsl,ldap,dom"
```

Comment:
Do we really need all of these just to run Psalm?

Reply:
No, but if we dont install `memcache` then psalm will complain about `\Memcache` class is not found. So psalm will run just fine without these.

Reply:
I see.